terasoluna-gfw-starters/terasoluna-gfw-starter-jodatime/pom.xml

```diff
@@ -7,7 +7,7 @@
 		<groupId>org.terasoluna.gfw</groupId>
 		<artifactId>terasoluna-gfw-starters</artifactId>
 		<version>5.2.0-SNAPSHOT</version>
-		<relativePath>../terasoluna-gfw-starters/pom.xml</relativePath>
+		<relativePath>../../terasoluna-gfw-starters/pom.xml</relativePath>
```

Comment:
@ikeyat 
By using ```<relativePath>../../terasoluna-gfw-starters/pom.xml</relativePath> ``` , I have tried to follow the approach used in the artifact like ```terasoluna-gfw-codepoints-jisx0201``` .

However, I feel that the value ``` ../pom.xml``` and ```../../terasoluna-gfw-starters/pom.xml``` used in the relativePath tag of each of the starter pom refers to the ```pom``` from ```terasoluna-gfw-starters```